spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { FactoryBot.build(:user, last_name: 'Doe', first_name: 'John') }
```

Comment:
FactoryBot can be omitted: just `build(:user, ...)`

also, use the keyword `subject` to define the subject:
```
subject { build(:user, last_name: 'Doe', first_name: 'John') }
```